Src/AutoFixture.xUnit.net2.UnitTest/AutoFixture.xUnit.net2.UnitTest.csproj

```diff
@@ -93,6 +94,9 @@
   <ItemGroup>
     <None Include="packages.config" />
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
```

Comment:
@ecampidoglio, you're right, this line is automatically added every time I run tests in VS Test Explorer. I shouldn't have mixed it with all the other changes, so I'm fine with removing it from the commit.